payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayPaymentMethodLauncher.kt

```diff
@@ -271,6 +273,12 @@ class GooglePayPaymentMethodLauncher @AssistedInject internal constructor(
          */
         var isEmailRequired: Boolean = false,
 
+        /**
+         * Shipping address collection configuration.
+         */
+
+        var shippingAddressConfig: ShippingAddressConfig = ShippingAddressConfig(),
```

Comment:
One nit about formatting. Also, why is this not a `val`? (Same applies to other fields in this object)

```suggestion
        /**
         * Shipping address collection configuration.
         */
        var shippingAddressConfig: ShippingAddressConfig = ShippingAddressConfig(),
```

Reply:
Oops should be val, mistype